pkg/rekor/identity.go

```diff
@@ -74,19 +74,21 @@ type MonitoredValues struct {
 	Subjects []string `yaml:"subjects"`
 }
 
-// IdentityEntry holds a certificate subject, issuer, and log entry metadata
+// IdentityEntry holds a certificate subject, issuer, OID extension and associated value, and log entry metadata
 type IdentityEntry struct {
-	CertSubject string
-	Issuer      string
-	Fingerprint string
-	Subject     string
-	Index       int64
-	UUID        string
+	CertSubject    string
+	Issuer         string
+	Fingerprint    string
+	Subject        string
+	Index          int64
+	UUID           string
+	OIDExtension   asn1.ObjectIdentifier
```

Comment:
Can we rebase, since this was this already done in #445?

Reply:
just rebased!